launchUI.py

```diff
@@ -1616,19 +1619,19 @@ def create_param_pannel(master):
     place_fu_options(fuConfigPannel)
 
     xbarConfigPannel = tkinter.LabelFrame(paramPannel, text='Tile 0 crossbar outgoing links', bd=BORDER, relief='groove')
-    xbarConfigPannel.grid(columnspan=4, row=7, column=0,rowspan=2)
+    xbarConfigPannel.grid(columnspan=4, row=7, column=0,rowspan=2,sticky="nsew")
     widgets["xbarConfigPannel"] = xbarConfigPannel
-    place_xbar_options(xbarConfigPannel)   
+    place_xbar_options(xbarConfigPannel)
 
     spmConfigPannel = tkinter.LabelFrame(paramPannel, text='Data SPM outgoing links', bd=BORDER, relief='groove')
-    spmConfigPannel.grid(row=7,column=0,rowspan=2,columnspan=4)
+    spmConfigPannel.grid(row=7,column=0,rowspan=2,columnspan=4,sticky="nsew")
     widgets["spmConfigPannel"] = spmConfigPannel
     for i in range(3):
         spmConfigPannel.rowconfigure(i,weight=1)
     
-    spmConfigPannel.columnconfigure(0,weight=2)
+    spmConfigPannel.columnconfigure(0,weight=1)

```

Comment:
Can you add a comment here about what are the `columnconfigure` used for?

Reply:
Yes. Here the columnconfigure means that the spmConfigPannel will be evenly divided into three columns, and each column will fill the corresponding grid space.